functions/Test-DbaDbQueryStore.ps1

```diff
@@ -120,7 +120,7 @@ function Test-DbaDbQueryStore {
                 }
                 'Microsoft.SqlServer.Management.Smo.Database' {
                     Write-Message -Level Verbose -Message "Processing Database through InputObject"
-                    $dbDatabases = $input | Where-Object { -not $ExcludeDatabase }
+                    $dbDatabases = $input | Where-Object { $_.Name -notin $ExcludeDatabase }
```

Comment:
`dbDatabases = $input | Where-Object Name -notin $ExcludeDatabase`

is prettier and faster 💅🏼